File: LeetCode/2706. Buy Two Chocolates/Solution.py
class Solution:
    def buyChoco(self, prices: list[int], money: int) -> int:
        firstMin, secondMin = 101, 101

        for price in prices:
            if price < firstMin:
                secondMin = firstMin
                firstMin = price
            elif price < secondMin:
                secondMin = price

        if money >= (firstMin + secondMin):
            money -= firstMin + secondMin

        return money

File: LeetCode/2706. Buy Two Chocolates/Solution.cpp
class Solution {
public:
    int buyChoco(vector<int>& prices, int money) {
        int firstMin = 101, secondMin = 101;

        for (int i = 0; i < prices.size(); i++) {
            if (prices[i] < firstMin) {
                secondMin = firstMin;
                firstMin = prices[i];
              
            } else if (prices[i] < secondMin) {
                secondMin = prices[i];
            }
        }

        if (money >= firstMin + secondMin) {
            return money - firstMin - secondMin;
        } else {
            return money;
        }

    }
};
